libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -255,7 +269,18 @@ void AP_Follow::handle_msg(const mavlink_message_t &msg)
 
         _target_location.lat = packet.lat;
         _target_location.lng = packet.lon;
-        _target_location.alt = packet.alt / 10;     // convert millimeters to cm
+        
+        // select altitude source based on FOLL_ALT_TYPE param 
+        if (_alt_type == AP_FOLLOW_ALTITUDE_TYPE_RELATIVE) {
+            // relative altitude
+            _target_location.alt = packet.relative_alt / 10;        // convert millimeters to cm
```

Comment:
better to multiply by 0.1f? 
to be consistent with velocity calculation

Reply:
Since ```_target_location.alt``` is of type ```int32_t```, I think we should avoid implicit type conversion here.